src/backends/torch/torchlib.cc

```diff
@@ -900,21 +977,22 @@ namespace dd
 
     bool lstm_continuation = false;
     TInputConnectorStrategy inputc(this->_inputc);
+    if (_module._native != nullptr)
+      _module._native->update_input_connector(inputc);
+
     TOutputConnectorStrategy outputc(this->_outputc);
-    ;
     try
       {
         inputc.transform(ad);
-        if (_module._native)
+        if (_module._graph)
```

Comment:
Similar to my previous remark, I'd see this block of code going first into `TorchModule` with a function call replacing the block.

Next I'd envison that anything that deals with `graph` internals remains within `graph`, typically the `lstm_continues` stuff, i.e. as far as possible by design.

Reply:
ok, done